healthcheck/healthcheck.py

```diff
@@ -116,18 +117,30 @@ class FilesExistHealthCheck(ListHealthCheck):
     """Fails if at least one of passed files doesn't exist."""
 
     def check_item(self, filename):
-        file_exists = os.path.isfile(filename)
-        details = {filename: 'exists' if file_exists else 'NO SUCH FILE'}
-        return file_exists, details
+        try:
+            ok = file_exists(filename)
+        except OSError as e:
+            ok = False
+            description = 'ERROR: ' + str(e.strerror)
```

Comment:
PyCharm annotates `strerror` as an `int` and so gives me a type warning without the `str()`. This is primarily to squash that. It also makes things more robust if `strerror` happens to not be a string, e.g. if it's `None`, which [the docs](https://docs.python.org/2/library/exceptions.html#exceptions.EnvironmentError) say it potentially could be.